game/models.py

```
@@ -518,7 +518,7 @@ class ActiveGame(models.Model):
 
     class Meta:
         indexes = [
-            models.Index(fields=["status", "last_active"]),
+            models.Index(fields=["status", "last_activity_at"]),
         ]
```

Ruff (0.15.20)

[warning] Mutable default value for class attribute

(RUF012)